pygments/lexers/ul4.py

```diff
@@ -0,0 +1,261 @@
+"""
+    pygments.lexers.ul4
+    ~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the UL4 templating language.
+
+    More information: https://python.livinglogic.de/UL4.html
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments import lexer, token
+from pygments.lexers import web, python
```

Comment:
`Contributing.md` recommends to import the names that you need.

Reply:
OK, I (mis-)interpreted that as an advice against using `from pygments.token import *`.

Fixed in https://github.com/doerwalter/pygments/commit/2476dbd94ebfdd3cb36e9db9f0fa81a1ea8a6fce